commands/exportcommands.py

```
@@ -334,100 +301,148 @@ def _apply_content_filtering(self, file_path):
+            # Strategy 1: Try to get latest git tag that looks like a version
+            result = subprocess.run(['git', 'describe', '--tags', '--abbrev=0'], 
+                                  capture_output=True, text=True, cwd=self.project_root)
+            if result.returncode == 0:
+                tag = result.stdout.strip()
+                # Extract version from tag (handle v1.2.3 or 1.2.3 formats)
+                version_match = re.search(r'v?(\d+\.\d+\.\d+)', tag)
+                if version_match:
+                    version = version_match.group(1)
+                    print(f"   📋 Found git tag version: {version}")
+                    return self._increment_version(version)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)